### PR TITLE
Initial Authentication Service Structure

### PR DESCRIPTION
**Pull request by tphalp**
Adds basic structure for the Authentication Service.

Resolves UCP-120

### CODE REVIEW

File: apps/authentication-service/src/tokens/tokens.model.ts
Comment (tphalp) [inline apps/authentication-service/src/tokens/tokens.model.ts:1]: Placeholder for next phase

File: apps/authentication-service/src/tokens/tokens.router.ts
Comment (tphalp) [inline apps/authentication-service/src/tokens/tokens.router.ts:1]: Placeholder for next phase

File: apps/authentication-service/src/tokens/tokens.service.ts
Comment (tphalp) [inline apps/authentication-service/src/tokens/tokens.service.ts:1]: Placeholder for next steps

**Review by Craiting — APPROVED** [review]

File: apps/authentication-service/src/middleware/auth0.middleware.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import * as dotenv from "dotenv";
+import { auth } from "express-oauth2-jwt-bearer";
+
+dotenv.config();
+
+export const validateAccessToken = auth({
```
Comment (wesrisenmay-mx) [inline apps/authentication-service/src/middleware/auth0.middleware.ts:6]: Is this not being used?

File: apps/authentication-service/src/middleware/auth0.middleware.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  audience: process.env.AUTH0_AUDIENCE,
+  issuerBaseURL: `https://${process.env.AUTH0_DOMAIN}`,
```
Comment (wesrisenmay-mx) [inline apps/authentication-service/src/middleware/auth0.middleware.ts:8]: I'd probably have an environment.ts file somewhere that pulls everything from process.env

File: apps/authentication-service/src/middleware/auth0.middleware.ts (hunk 1 above)
Comment (tphalp) [reply apps/authentication-service/src/middleware/auth0.middleware.ts:6]: Yeah, not yet. I got tired of waiting for an approval, and let some of my next code sneak in.

**Pull request merged**